Client/Frontend/Home/HomepageViewModel.swift

```diff
@@ -80,6 +80,7 @@ class HomepageViewModel: FeatureFlaggable {
     var shownSections = [HomepageSectionType]()
     weak var delegate: HomepageViewModelDelegate?
     private var wallpaperManager: WallpaperManager
+    private var siteImageHelper: SiteImageHelperProtocol
```

Comment:
Why do we save it in HomepageViewModel? If we save it maybe we should pass it down to child view models that currently created their own siteImageHelper (history highlights, jump back in, recently saved, top sites).

Reply:
I have no idea why I stored this 🙈